srcpkgs/youtube-dl/template

```diff
@@ -1,21 +1,21 @@
 # Template file for 'youtube-dl'
 pkgname=youtube-dl
-version=2019.07.27
+version=2019.08.02
 revision=1
 archs=noarch
 wrksrc="$pkgname"
-build_style=python-module
+build_style=python3-module
 pycompile_module="youtube_dl"
-hostmakedepends="python python3"
-depends="python"
-short_desc="CLI program to download videos from YouTube and other sites (Python2)"
+hostmakedepends="python3"
+depends="python3"
+short_desc="CLI program to download videos from YouTube and other sites"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="Unlicense"
 homepage="https://rg3.github.io/youtube-dl/"
 changelog="https://raw.githubusercontent.com/rg3/youtube-dl/master/ChangeLog"
 distfiles="https://yt-dl.org/downloads/${version}/${pkgname}-${version}.tar.gz"
-checksum=cdfc691ca9dcc63addb52dbbadb1b9e86325dd2d53d38ac844d504f482468bcc
-alternatives="youtube-dl:youtube-dl:/usr/bin/youtube-dl2"
+checksum=573c37add3e6107bf0c3c8ce43443bb788957d7bfb5e736cee6b32e3b2342b80
+replaces="python3-youtube-dl>=0"
```

Comment:
This is problematic. If someone has python3-youtube-dl installed, it will pull in youtube-dl while removing the empty metapackage. This is fine however it will leave youtube-dl as an automatically installed package no other package depends on so it will be subject to removal by xbps-remove -o. I think it is better to leave the meta package installed in these cases.

Reply:
just remove the replaces. :+1: